Source/WebCore/page/scrolling/ScrollingTreeScrollingNode.h

```diff
@@ -132,6 +132,7 @@ class WEBCORE_EXPORT ScrollingTreeScrollingNode : public ScrollingTreeNode {
     OverscrollBehavior verticalOverscrollBehavior() const { return m_scrollableAreaParameters.verticalOverscrollBehavior; }
     
     virtual String scrollbarStateForOrientation(ScrollbarOrientation) const { return ""_s; }
+    IntPoint lastKnownMousePosition() const;
```

Comment:
Again, confusion about coordinate systems.

Reply:
Renamed to be raw mouse position.